components/brave_wallet/browser/network_manager_unittest.cc

```diff
@@ -0,0 +1,1050 @@
+
+#include "brave/components/brave_wallet/browser/network_manager.h"
+
+#include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/containers/flat_set.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/string_util.h"
+#include "base/test/scoped_feature_list.h"
+#include "base/test/values_test_util.h"
+#include "base/values.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_prefs.h"
+#include "brave/components/brave_wallet/browser/pref_names.h"
+#include "brave/components/brave_wallet/browser/test_utils.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+#include "brave/components/brave_wallet/common/common_utils.h"
+#include "brave/components/brave_wallet/common/features.h"
+#include "brave/components/brave_wallet/common/test_utils.h"
+#include "components/prefs/scoped_user_pref_update.h"
+#include "components/sync_preferences/testing_pref_service_syncable.h"
+#include "testing/gmock/include/gmock/gmock.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "url/origin.h"
+
+using base::test::ParseJsonDict;
+using testing::Contains;
+using testing::ElementsAreArray;
+using testing::Eq;
+using testing::Not;
+
+namespace brave_wallet {
+
+// DEPRECATED 01/2024. For migration only.
+std::string GetSolanaSubdomainForKnownChainId(const std::string& chain_id);
+std::string GetFilecoinSubdomainForKnownChainId(const std::string& chain_id);
+std::string GetBitcoinSubdomainForKnownChainId(const std::string& chain_id);
+std::string GetZCashSubdomainForKnownChainId(const std::string& chain_id);
+std::string GetKnownEthNetworkId(const std::string& chain_id);
+std::string GetKnownSolNetworkId(const std::string& chain_id);
+std::string GetKnownFilNetworkId(const std::string& chain_id);
+std::string GetKnownBtcNetworkId(const std::string& chain_id);
+std::string GetKnownZecNetworkId(const std::string& chain_id);
+
+namespace {
+
+// void UpdateCustomNetworks(PrefService* prefs,
+//                           const std::vector<base::Value::Dict>& values,
+//                           brave_wallet::mojom::CoinType coin) {
+//   ScopedDictPrefUpdate update(prefs, kBraveWalletCustomNetworks);
+//   base::Value::List* list = update->EnsureList(GetPrefKeyForCoinType(coin));
+//   list->clear();
+//   for (auto& it : values) {
+//     list->Append(it.Clone());
+//   }
+// }
+
+// void UpdateCustomNetworks(PrefService* prefs,
+//                           const base::Value::Dict& value,
+//                           brave_wallet::mojom::CoinType coin) {
+//   std::vector<base::Value::Dict> values;
+//   values.push_back(value.Clone());
+//   UpdateCustomNetworks(prefs, values, coin);
+// }
+
+// std::vector<brave_wallet::mojom::NetworkInfoPtr> GetAllEthCustomChains(
+//     NetworkManager* network_manager) {
+//   return network_manager->GetAllCustomChains(
+//       brave_wallet::mojom::CoinType::ETH);
+// }
```

Comment:
Why leave these?

Reply:
cleaned this: https://github.com/brave/brave-core/pull/24365/commits/90f3a806f44fb0020b62971c6a305622a0c6a362